conans/client/build/meson.py

```diff
@@ -164,7 +164,11 @@ def configure(self, args=None, defs=None, source_dir=None, build_dir=None,
             build_type
         ])
         command = 'meson "%s" "%s" %s' % (source_dir, self.build_dir, arg_list)
-        with environment_append({"PKG_CONFIG_PATH": pc_paths}):
+        envs = {"PKG_CONFIG_PATH": pc_paths}
+        if self._append_vcvars and str.startswith(self.backend, "vs"):
```

Comment:
why not ``self.backend.startswith("vs")``? This is the usual syntax we use in the codebase.

Reply:
Yes, there is no difference between the two, then do as you say.

I do this just for the convenience of automatic coding.